### ISSUE TITLE
Build your own Multithreaded Web Server in Rust

### ISSUE THREAD

**Issue by Duckulus** [codecrafters-io/build-your-own-x#799]
### Main programming language
Rust

### Tutorial title
Building a Multithreaded Web Server

### Tutorial URL
https://doc.rust-lang.org/stable/book/ch20-00-final-project-a-web-server.html

### Category
* [ ] Web Server

**Comment by ranjansmygit** [comment]
Could you elaborate on issue, what's the issue ?

**Comment by malinjr07** [comment]
Yep. we can build our own web server to manage hosting if big cats gets greedy!
